lib/ansible/plugins/filter/b64decode.yml

```diff
@@ -5,12 +5,16 @@ DOCUMENTATION:
   short_description: Decode a base64 string
   description:
     - Base64 decoding function
-  positional: _input
+  positional: input
```

Comment:
This change is incorrect, it is following a convention that `_input` is named by not name addressable, you will see the same with `_raw`/`_raw_params`

Reply:
unsure, maybe we need to namespace them to plugin.option?

Reply:
The duplicate target error is because with the currently released antsibull-docs version `_input` is both a positional and the input parameter for this filter (the former because it is mentioned in `positional`, the latter because it's called `_input`). #77725 fixes that, but since it isn't clear whether that has a chance of being merged there's a fix in antsibull-docs (not released yet) that ignores `_input` in `positional`.